mcs/build/platforms/darwin.make

```diff
@@ -9,9 +9,9 @@ PLATFORM_RUNTIME = $(RUNTIME)
 PLATFORM_CORLIB = mscorlib.dll
 PLATFORM_TEST_HARNESS_EXCLUDES = NotOnMac,
 
-EXTERNAL_MCS = mcs
-EXTERNAL_MBAS = mbas
-EXTERNAL_RUNTIME = mono
+EXTERNAL_MCS := $(shell which mcs)
```

Comment:
Because if it's "=" then this is done lazily. If this were computed when another script had changed the PATH, then the external mono might end up pointing to something else. We'd have the same problem we have when using a relative path.